### PR TITLE
UofT-DSI | Deep Learning - Assignment 1

### PR DESCRIPTION
**Pull request by Franceschung**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Added code for assignment 1 and first week labs
## What did you learn from the changes you have made?
Deep learning
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
tried with a lot of hyperparameters setup
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
a lot, take time to compute with different setup
## How were these changes tested?
executed with google colab
## A reference to a related issue in your repository (if applicable)
https://keras.io/
## Checklist
- [X] I can confirm that my changes are working as intended

### CODE REVIEW

File: deep_learning/01_materials/labs/lab_1.ipynb
Comment (tianyi21) [inline deep_learning/01_materials/labs/lab_1.ipynb:1]: - b: Please always reinitialize your model with `model = Sequential() ...` to reset all model parameters in addition to `model.compile()`. Or, you are continuing the training from the previous trained state with a different model setup. You may check this with the accuracy of the first epoch.
- Impact of Initialization: You initialized the weights with `stddev=0.01` but `w.std()` gives `0.88359493`. Your implementation was correct here.

**Review by tianyi21 — APPROVED** [review]
Hi Oi Yee!

Thank you for submitting your DL assignments. Please refer to my comments attached to each file.

Your A1 and Lab 1-3 are now complete.

Thanks,
Tianyi [LS]
